Gemfile.base

```diff
@@ -63,3 +62,8 @@ gem 'bugsnag', '~> 6', require: nil
 gem 'yabeda-prometheus', '~> 0.5.0'
 gem 'async-redis', '~> 0.4.1'
 gem 'falcon', '~> 0.35'
+
+# Use a patched redis-rb that fixes an issue when trying to connect with
+# sentinels and avoids retrying calls when there's a timeout to prevent
+# duplicated commands. It's based on version 4.1.3.
+gem 'redis', git: 'https://github.com/3scale/redis-rb', ref: '35301b3d952975300a4cb30d408ae3b5969d0440'
```

Comment:
I prefer this to refer to a branch, then locking the revision in the lockfile.

Reply:
I just followed what we're doing for Puma, but I can change this, I see that in others we use branches.

Reply:
Use the example of resque - the puma one is wrong, it has a branch that we should use.

Reply:
Changed :+1: